docs/src/modules/components/AppNavDrawer.js

```diff
@@ -452,7 +454,28 @@ export default function AppNavDrawer(props) {
               metadata="MUI X"
               versionSelector={renderVersionSelector([
                 // DATA_GRID_VERSION is set from the X repo
-                { text: `v${process.env.DATA_GRID_VERSION}`, current: true },
+                {
+                  text: 'v6-alpha',
+                  ...(process.env.DATA_GRID_VERSION.startsWith('6')
+                    ? {
+                        text: `v${process.env.DATA_GRID_VERSION}`,
+                        current: true,
+                      }
+                    : {
+                        href: `https://next.mui.com${languagePrefix}/components/data-grid/`,
+                      }),
+                },
+                {
+                  text: 'v5',
+                  ...(process.env.DATA_GRID_VERSION.startsWith('5')
+                    ? {
+                        text: `v${process.env.DATE_PICKERS_VERSION}`,
```

Comment:
```suggestion
                        text: `v${process.env.DATA_GRID_VERSION}`,
```
Shouldn't it reference Data grid version?
In the linked demos I've looked currently both data-grid and date-pickers documentations list date-pickers version when navigating v5. 🤔

Reply:
Nice catch